src/components/Sidebar/index.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+
+import Box from '../Box';
+
+import './style.scss';
+
+const Sidebar = () => {
+  const currentUser = JSON.parse(localStorage.getItem('currentUser'));
+  const userImage = 'https://www.laprensa.com.ar/multimedios/imgs/123027_620.jpg';
+
+  if (!currentUser) {
+    return null;
+  }
+
+  return (
+    <div className="sidebar">
+      <Box>
```

Comment:
Can't we add the classname to the Box as a customClassName so we don't need the div?